webpack/JobWizard/JobWizardPageRerun.js

```diff
@@ -61,14 +85,22 @@ const JobWizardPageRerun = ({
         <React.Fragment>
           <Divider component="div" />
         </React.Fragment>
-        {!status || status === STATUS.PENDING ? (
+        {jobInvocationStatus === STATUS.ERROR && (
+          <Alert
+            ouiaId="job-wizard-alert-error"
+            variant="danger"
+            title={sprintf(errorMessage)}
+          />
```

Comment:
I think it would be better to have some red empty state like here: https://v4-archive.patternfly.org/v4/components/empty-state/design-guidelines#back-end-failure
With a button to create a job